### DISCUSSION TITLE
Login page is shown as restricted page

### DISCUSSION THREAD

**Discussion by rikss2** [community/community#126096] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I got the restricted page error page when trying to "star" a repository without being logged in. I sadly ran into a cookie issue possibly, where whichever GitHub page I tried opening: everything was restricted. Cookies are hard when it comes to Customer IAM. I would love to understand your authorization/session management which I could not decode as a standard token. Extra points there :3

I was finally able to bypass this via the support page being "gated content", which is unfortunate. _Try keeping a ticket intake for login issues, even for those who did not manage to log in._
You have a great self-service experience, and a nice, intuitive look and feel for customer identification, keep up the good work!

**Comment by playgude2** [comment]
response #2
